lib/docker.js

```diff
@@ -764,20 +774,40 @@ async function startInstallationContainer({ runtime, imageName, codeUri, targets
   }
 
   const codeMount = await resolveCodeUriToMount(codeUri, false);
-
   const installMounts = conventInstallTargetsToMounts(targets);
-
   const mounts = [codeMount, ...installMounts];
 
   await pullImageIfNeed(imageName);
 
   const envs = addInstallTargetEnv({}, targets);
-
   const opts = dockerOpts.generateInstallOpts(imageName, mounts, envs);
 
   return await startContainer(opts);
 }
 
+function displaySboxTips(runtime) {
+  console.log(yellow(`\nWelcom to fun sbox environment.\n`));
+  console.log(yellow(`You can install system dependencies in the following way:`));
```

Comment:
You can install system dependencies in the following way:
=>
You can install system dependencies like this:

Reply:
done